operations/itunestesters/README.md

```diff
@@ -0,0 +1,135 @@
+<p align="center">
+    <img src="internal/testflight_builds_page.png">
+</p>
+
+7. **Select 'App Store Connect Users'** from the right hand menu, you will be presented with a list of users that you saw previously in the 'Users and Access' section but who have already been invited to receive builds.
+<p align="center">
+    <img src="internal/testflight_appStoreUsers_page.png">
+</p>
+
+8. **Select the blue '+' icon** and you will see the following scrollable dialogue box, scroll through until you see the user that you want to add, **note - if they are not there, they haven't accepted the first email**. Once selected, **select the 'add' button**, a second email will be sent to the user and they will be invited to receive builds via testFlight (apple's test app distribution system, available on the appstore).
+<p align="center">
+    <img src="internal/internalTester_build_dialogue.png">
+</p>
+
+9. Once the user has downloaded the testFlight app and tapped the link in the email, they will then be able to access all existing and future builds of the app.
+
+## External Testers
+
+### **NOTE**
+External testers are added to external builds of the app, an external build is a specific build that needs to be sent for 'beta approval', this is a miniature review that apple does to ensure the app is stable and safe for use by testers as any build distributed this way be sent to up to 10,000 users.
+
+**The beta review process can take between 24-48 hours** and will only be available to external testers if it successfully passes this process.
+
+Another thing to mention is that only the first build of a version requires external review, subsequent builds of the same version will be automatically approved.
+
+For example:
+You submit version 11.1 build 11234 for external review, after 24 hours, the build is approved and your testers are happy finding bugs and raising issues. Your friendly neighbourhood developers lovingly resolve the issues and another build is uploaded to iTunesConnect, version 11.1 build 11235, as they share the same version number, build 11235 can be submitted for external review again and will automatically be approved.
+
+If there were substantial changes that required a version change, upping the version to 11.2 for example, this would then require the full external review process of 24-48 hours.
+
+### What is an external tester?
+An external tester is someone who will only want to receive select builds, either they do not need to see every build, or they are added to test at specific times, such as UAT (User Acceptance Testing) sessions or when requiring sign-off.
+
+### Who should be an external tester?
+A user should be added as an external tester if they are likely to report issues and distract from the app development process, in this case, it is best the user is added as an external so they see the best, curated builds that are in a better position to be demo'ed.
```

Comment:
> A user should be added as an external tester if they are likely to report issues and distract from the app development process, in this case, it is best

Not sure this makes sense. I think the intention is to say something like
"Some testers may give too much feedback and not understand that many development builds contain known bugs. By adding them as an external tester, you can control which builds they are asked to test, ensuring they only see curated builds that are in a better position to be demo'ed."